### PR TITLE
Note about empty tags

### PR DESCRIPTION
**Pull request by annstriganova**
Notes about empty tags search and timestamp field.

### CODE REVIEW

**Comment by annstriganova** [comment]
Notes about empty tags search and `timestamp` field.

File: rule-engine/functions-db.md [hunk 1]
```diff
@@ -326,6 +327,7 @@ The following matching rules apply:
 
 * Tags:
   * If the tags `t` argument is specified as `null` or empty string `''`, all tags are matched.
+  * To match records with empty tags use `'tags.isEmpty()=true'` in expression `p`.
```
Comment (rodionos) [inline rule-engine/functions-db.md]: Can we also use `tags.size() == 0`?

Comment (annstriganova) [reply rule-engine/functions-db.md]: Yes, we can. Add to documents or replace `isEmpty()=true` with `size() = 0`?

**Comment by axibase-git** [comment]
Include both options.

On Mar 3, 2018, at 5:23 PM, Anna Striganova <notifications@github.com> wrote:

@annstriganova commented on this pull request.

In rule-engine/functions-db.md <https://github.com/axibase/atsd/pull/441#discussion_r172018369>:

> @@ -326,6 +327,7 @@ The following matching rules apply:
 
 * Tags:
   * If the tags `t` argument is specified as `null` or empty string `''`, all tags are matched.
+  * To match records with empty tags use `'tags.isEmpty()=true'` in expression `p`.
Yes, we can. Add to documents or replace isEmpty()=true with size() = 0?

**Comment by raipc** [comment]
Another way: `tags = empty`

**Pull request merged**